app/components/MigrateVault/index.js

```diff
@@ -19,17 +19,19 @@ const MigrateVault = ({ vaultAddress }) => {
 
   return (
     <Tooltip title="Migrate your balance to v2 Vault">
-      <ButtonFilled
-        onClick={() =>
-          dispatch(migrateVault({ vaultContract, trustedMigratorContract }))
-        }
-        color="primary"
-        disabled={!vaultContract || !trustedMigratorContract}
-        disabledTooltipText="Connect your wallet to migrate vault"
-        showTooltipWhenDisabled
-      >
-        Migrate all
-      </ButtonFilled>
+      <>
```

Comment:
Work around for forward ref material-UI tooltip issue. Button filled implements material-UI using `withStyled` (as documents suggest) but Tooltip doesn't like nesting of `withStyled` elements.